x-pack/legacy/plugins/apm/public/components/app/ErrorGroupOverview/List/index.tsx

```diff
@@ -191,28 +137,24 @@ export class ErrorGroupList extends Component<Props, State> {
         render: (value?: number) =>
           value ? moment(value).fromNow() : NOT_AVAILABLE_LABEL
       }
-    ];
-
-    return (
-      <EuiBasicTable
-        noItemsMessage={i18n.translate('xpack.apm.errorsTable.noErrorsLabel', {
-          defaultMessage: 'No errors were found'
-        })}
-        items={paginatedItems}
-        columns={columns}
-        pagination={{
-          pageIndex: this.state.page.index,
-          pageSize: this.state.page.size,
-          totalItemCount: this.props.items.length
-        }}
-        sorting={{
-          sort: {
-            field: sortField || 'latestOccurrenceAt',
-            direction: sortDirection || 'desc'
-          }
-        }}
-        onChange={this.onTableChange}
-      />
-    );
-  }
-}
+    ],
+    [serviceName]
+  );
+
+  return (
+    <ManagedTable
```

Comment:
Is the list still sorted on the backend? If so, do we need the ManagedTable (isn't that just sorting it on the client?)

Reply:
It needs to be sliced, something the ManagedTable does as well. I thought it would make more sense to just use ManagedTable with an explicit `sortItems` prop rather than copying a lot of the URL logic used in ManagedTable to this file as well. It's not very pretty, definitely open for suggestions.

Reply:
That's sounds good. Just double checking :)